### ISSUE TITLE
[Accessibility] Smart Filter Bar's refresh button & filter checkbox has no aria-label; Filter name are read repeatly

### ISSUE THREAD

**Issue by MIAeh** [SAP/fundamental-ngx#10131]
#### Is this a bug, enhancement, or feature request?
Bug

#### Briefly describe your proposal.

1. For the refresh button
<img width="1304" alt="Screenshot 2023-07-06 at 12 22 51" src="https://github.com/SAP/fundamental-ngx/assets/26197222/ac5fe777-e7bd-4bc1-922b-5d6f440f7444">

Visible label should be provided for the ‘Refresh’ button within the ‘Staged Products’ tool bar region.  
All UI elements should have visible and meaningful label provided and it should associated with the UI. Meaningful label supports the user in entering data correctly.

2. For the filters dialog - checkbox group
<img width="452" alt="image" src="https://github.com/SAP/fundamental-ngx/assets/26197222/3c4a4152-91b0-4bc8-a9e8-4fa81c24efd8">
On accessing the checkboxes within the ‘Filters’ dialog, the associated labels with the checkboxes are not getting announced.

3. For the repeated results read out:
<img width="452" alt="image" src="https://github.com/SAP/fundamental-ngx/assets/26197222/f589090a-a931-4f55-a110-491d441c638a">
Only relevant information about the UI should be communicated to the user. Additional information should be avoided.]

4. The 'Filters' dialog doesn't contain a necessary identifier
![Screenshot 2023-07-11 at 13 52 00](https://github.com/SAP/fundamental-ngx/assets/26197222/662509f6-b5c3-4fb1-9f61-595a28ef4d27)
![Screenshot 2023-07-11 at 13 51 54](https://github.com/SAP/fundamental-ngx/assets/26197222/a00c0af4-5bee-4855-8bee-71d8bba62d05)

#### Which versions of Angular and Fundamental Library for Angular are affected? (If this is a feature request, use current version.)
The lastest version also contains this issue.

#### If this is a bug, please provide steps for reproducing it.
Any sample in doc page could reproduce.

#### Please provide relevant source code if applicable.
https://github.com/SAP/fundamental-ngx/blob/main/libs/platform/src/lib/smart-filter-bar/smart-filter-bar.component.html#L26

**Comment by mikerodonnell89** [comment]
This issue is out of date - all of these appear to be working as expected
